Adyen/Assets/Generated/ADYLocalizationKey.swift

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+// This file is autogenerated. Please do not modify it.
+
+/// :nodoc:
+public struct ADYLocalizationKey {
```

Comment:
also what's up with the `ADY` prefix, its not necessary at all, its not objective c :-) .

Reply:
The function `ADYLocalizedString` has it, so i thought there was some magic to it. I can make it `enum LocalizationKey: String`